### ISSUE TITLE
[BUG] instant crash before game in starbase arena starts => counts as loss => 1k rating lost

### ISSUE THREAD

**Issue by TaipionsSalemClient** [PavelZinchenko/event-horizon-frontier-builds#390]
First off, I can't even start to describe how... for the love of god... horrible, fucked up and incredibly bad the whole arena rating, rating calculations, matchmaking and win/loss determination is...

That put aside,
here's the BUG:

- was doing starbase arena fights at the "Starbase Battle" node
- "worthy" opponent pops up with 100k defence rating (me being over 103k attack rating means I have to only take enemies above like 98500 or so, see first paragraph for what I think about this)
- click on the attack icon
- I am thrown back to the star map, no fight started or happened
- looked around, nothing seemed to have happend
- continued to do fights
- one or two fights later I noticed: **I LOST 1k RATING**

conclusion => the fight somehow crashed before it started, still counted as a loss to me

please fix this
(and all the other horrible shit that is arena matchmaking, rating and so on...)

Thank You!

**Comment by TaipionsSalemClient** [comment]
it would also help, to make the arena win/loss determination a little more sophisticated,
instead of setting the point (presumably) on the button press to start the fight,
it should be after the player successfully loaded the battle,

also punishing players for game crashes is a really stupid idea,
you can already skip opponents (which still costs fight-tokens, which is again super stupid),
so why punish players for this?!

not to even begin with stating how this lost me 1k rating which is about 300 successful fights against 100k enemies at this point...

**Comment by SuplexCity25284** [comment]
Yeah, I have to agree with you.  If the game glitches out, and crashes, neither player should get losses for the system breaking down.  Being punished for that is insane.
